artemis-server/src/main/java/org/apache/activemq/artemis/spi/core/security/jaas/AuthenticatorAdapter.java

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
think javadoc is ok for the description, why not?

Reply:
Because this isnt the description, it isnt actually doc, its a licence header.  Its a waste of time for the javadoc processor to even be looking at it. Its formatted differently than it should be (has tags, and changed indent). Its inconsistent with the rest of the codebase.

Reply:
aah, ok, was looking at the wrong place. see the extra * in the licence template. sorted. thanks